docs/src/pages/components/data-grid/filtering/ServerFilterGrid.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  ColDef,
+  DataGrid,
+  FilterModelParams,
+  RowModel,
+} from '@material-ui/data-grid';
+import * as React from 'react';
```

Comment:
We try to import React first. I believe developers are used to this convention and unconsciously have learn to ignore this import.

```suggestion
import * as React from 'react';
import {
  ColDef,
  DataGrid,
  FilterModelParams,
  RowModel,
} from '@material-ui/data-grid';
```